app/controllers/application_controller.rb

```diff
@@ -470,6 +471,15 @@ def fetch_translations
     WebTranslateIt.fetch_translations
   end
 
+  def check_http_auth
+    return true unless APP_CONFIG.use_http_auth
```

Comment:
@rap1ds Good point. Interestingly, this works at the moment. I wonder if conversion to boolean happens somewhere in the middle by the configurator.